README.md

```diff
@@ -194,17 +192,73 @@ crafts = [
 _The first craft requires a stone axe to craft the item referenced "1", the second craft enable us to cook a cooked_porkchop into a coal, 
 and the third one is used to craft the item referenced "2" with three sticks aligned in a vertical centered line (notice the whitespaces before and after the "a")_
 
+### Affix
+
+Affix are modifiers who can be applied to item. They are pre-registered in pools (not items pools :wink:).
+The root is `affixes`, it contains a list of affix groups. a group is categorized by is `group` name : it's a schematic name only used for configuration like
+damage, speed, balanced, only_negative, etc...  
+   
+ And then the multiples `tiers` of each affix.
+ A tier is like a power level. `__class__` is the implementation of the affix, you can use your own
+ (leave it as is if you don't know what that mean). 
+   
+   
+ You can set a `prefix` and `suffix` who modify the name item. Don't leave space before and after,
+ these arguments they will be set afterward.  
+ The `probability` is the chance to chose this tiers more than another. the less it is, the less chance you have
+ to get this tier. Be careful to stay under the probability of one otherwise the tiers higher than one will 
```

Comment:
Si tu parles de math met en chiffre (one -> 1)  
Met aussi une virgule apres le 1, en anglais on privilegie les phrases courtes et on les separe facilement plutot que de faire des phrases complexes  
Ca fait peut etre plus joli et clair comme cela: `the tier which total probability exceeds 1 will never get picked`